src/components/date-picker-day/date-picker-day.scss

```diff
@@ -129,57 +129,93 @@
   }
 }
 
-:host([start-of-range]) :not(.calcite--rtl),
-:host([end-of-range]) .calcite--rtl {
-  .day-wrapper {
-    box-shadow: inset 4px 0 var(--calcite-ui-foreground-1);
+// Range start
```

Comment:
Let's drop these comments since they're not adding any more info than what the selector is already expressing.

Reply:
Yep, sorry that was for me to keep track while working.